test/test_pghoard.py

```diff
@@ -524,7 +524,28 @@ def test_startup_walk_for_missed_uncompressed_files(self):
         self.pghoard.startup_walk_for_missed_files()
         assert self.pghoard.compression_queue.qsize() == 2
         assert self.pghoard.transfer_queue.qsize() == 0
+    
+    def test_startup_walk_for_missed_uncompressed_files_timeline(self):
+        compressed_wal_path, _ = self.pghoard.create_backup_site_paths(self.test_site)
+        uncompressed_wal_path = compressed_wal_path + "_incoming"
+        with open(os.path.join(uncompressed_wal_path, "00000002.history"), "wb") as fp:
```

Comment:
I'd use a `with tempfile.TemporaryDirectory() as dir` so the file will be deleted at the end of the test :)

Reply:
Those files are created in `/tmp/` and it's taken care by teardown in `base.py`:

```
    def teardown_method(self, method):  # pylint: disable=unused-argument
        rmtree(self.temp_dir)
```